app/build.gradle.kts

```diff
@@ -5,12 +5,12 @@ plugins {
 
 android {
     namespace = "com.kevo.displaydemo"
-    compileSdk = 31
+    compileSdk = 33
```

Comment:
if this version bump isn't absolutely necessary, I'd recommend keeping this in sync with the POS codebase, to make it a more seamless transition (one of the objectives was to emulate the POS codebase ecosystem, so we could utilize these components with minimal headaches)